inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -139,7 +139,7 @@ public function migrate_attachment_to_s3( $args, $args_assoc ) {
 	 * Create an AWS IAM user for S3 Uploads to user
 	 *
 	 * @subcommand create-iam-user
-	 * @synopsis --admin-key=<key> --admin-secret=<secret> [--username=<username>]
+	 * @synopsis --admin-key=<key> --admin-secret=<secret> [--username=<username>] [--automatic]
```

Comment:
What about `--format=json` instead of `--automatic`, for better parity with other WP-CLI commands?